wcag-ruleset-review-process.md

```diff
@@ -1,77 +1,89 @@
 
-## Orphaned Rules
+Wherever possible, the ACT Task Force will work with the ACT Rules Community Group to come up with a joint answer to the questions. If the comment is about a proposed rule, or a proposed change to a rule a public issue is created, and an answer is sent referencing the issue. If the comment is on an approved rule and changes are necessary, a liaison is picked, and the change request is handled as survey feedback, starting in the [validation](#4-validation) step. 
 
-The ACT Task Force will send a request for change to a rule provider as a result of either public feedback or an annual review. If the rule provider does not respond within two months, the ACT Task Force will assume the rule is no longer actively maintained. The ACT Task Force will send at least 2 reminders before concluding the rule is not maintained. If a rule is not maintained, the ACT Task Force will look for another organization who might be willing to maintain the rule. If the original rule provider wishes to resume maintenance of the rule, this will only be accepted with the permission of the active rule provider.
+For the ACT Task Force agenda, public feedback will be prioritised over review of proposed rules. This ensures responsiveness to the wider community. The ACT Task Force should respond to questions within 5 work days. This may be a reply which indicates when an answers is expected.
 
-If no one can be found to maintain a rule within a month, the rule will be labelled as "outdated". Outdated rules will be removed from the ruleset after 1 year.
+## Annual Review
 
-## Rule Provider Questions
+Since technologies and standards frequently change, rules require regular maintenance. To ensure quality over time, the ACT Task Force will review every **approved rule** in the WCAG Ruleset at least once a year, to ensure it is still current and correct. If any changes are deemed necessary, a liaison is picked and the change request is handled as survey feedback, starting in the [validation](#4-validation) step. 
 
-Often when writing rules, rule providers have questions about the intended meaning of WCAG success criteria. The ACT Task Force will keep track of such questions coming from rule providers using GitHub issues and will work with the AG Working Group to come up with answers to them. Response to questions is subject to the availability of AG, therefore, so there is no target time frame for a response.
+Every time a rule is [validated](#4-validation), it is checked to make sure it is up to date. This counts as an annual review, so that only rules that have not been surveyed in 12 months need to go through a separate review. If a rule has not been reviewed for 12 months, an ACT facilitator opens an issue for it, and assigns it to that rule's liaison. The liaison has 10 work days to review if the rule needs to be updated. If the rule needs to be updated, the liaison has another 10 work days to create a [draft proposal](#1-draft-proposal).
```

Comment:
I'm not sure who the "rule's liaison" exactly is (since it is different from the ACT facilitator).
It is a bit of a strong requirement to give people a 10 work days deadline.
At https://github.com/w3c/wcag-act/pull/515/files#diff-1c6fbe9f1dcc2b7aeae3ce3d63c69e16cb5039c91b5b49a5ca0b8516897de840R80 it is a global deadline for the TF to answer, which makes sense, but here it seems to be an individual deadline. The way this sentence is written, it feels like  "that rule's liaison" is a well defined pre-existing person (the CG member who sent the rule to the TF, maybe?) If so, it is a strong requirement to ask them to be able to act within 10 work days at some unspecified point in the future.

Reply:
Liaisons can change. If whoever's the current liaison can't do it in 10 days, someone else needs to take over.